features/collection/components/FilterList/index.tsx

```diff
@@ -29,7 +29,145 @@ const FACET_BLOCKLIST = [
+   const [state, send] = useMachine(
+      createVirtualAccordionMachine<Facet>({
+         items: facets,
+         areRefined,
+         sizeMap: {},
+         expandedItemsIds: [],
+         toggledItemId: undefined,
+         toggledItemDelta: undefined,
+      }),
+      {
+         actions: {
+            setItemSize: assign((ctx, event) => {
+               return produce(ctx, (draft) => {
+                  if (event.type === 'ITEM_SIZE_UPDATED') {
+                     const index = draft.items.findIndex(
+                        (i) => i.name === event.id
+                     );
+                     if (index >= 0) {
+                        if (event.id === draft.toggledItemId) {
+                           draft.toggledItemDelta =
+                              draft.sizeMap[event.id] - event.size;
+                        }
+                        draft.sizeMap[event.id] = event.size;
+                        if (listRef.current) {
+                           listRef.current.resetAfterIndex(index);
+                        }
+                     }
+                  }
+               });
+            }),
+            setItems: assign((ctx, event) => {
+               return produce(ctx, (draft) => {
+                  if (event.type === 'ITEMS_CHANGED') {
+                     draft.items = event.items;
+                  }
+               });
+            }),
+            toggleItem: assign((ctx, event) => {
+               return produce(ctx, (draft) => {
+                  if (event.type === 'TOGGLE_ITEM') {
+                     if (draft.expandedItemsIds.includes(event.id)) {
+                        draft.expandedItemsIds = draft.expandedItemsIds.filter(
+                           (id) => id !== event.id
+                        );
+                     } else {
+                        draft.expandedItemsIds.push(event.id);
+                     }
+                     draft.toggledItemId = event.id;
+                  }
+               });
+            }),
+         },
+      }
+   );
+
+   React.useEffect(() => {
+      if (state.value === 'toggleItemAnimation') {
+         const timeoutId = setTimeout(() => {
+            send({
+               type: 'TOGGLE_ITEM_ANIMATION_END',
+            });
+         }, TOGGLE_ANIMATION_DURATION_MS);
+         return () => {
+            clearTimeout(timeoutId);
+         };
+      }
+   }, [send, state.value]);
+
+   React.useEffect(() => {
+      if (!isEqual(state.context.items, facets)) {
+         send({
+            type: 'ITEMS_CHANGED',
+            items: facets,
+            areRefined,
+         });
+      }
+   }, [areRefined, facets, send, state.context.items]);
+
+   const data = React.useMemo<ItemData>(() => {
+      return [state, send];
+   }, [state, send]);
+
+   const getSize = React.useCallback(
+      (index: number): number => {
+         return state.context.sizeMap[state.context.items[index].name] || 41;
```

Comment:
Can we make 41 a constant with a descriptive name? Seems like we use it in a few places.

Reply:
You're right, seems kind of a magic number.. It is actually the estimated default height of the row, so we could use something like `ROW_ESTIMATED_HEIGHT`